arches/app/utils/index_database.py

```diff
@@ -605,3 +607,47 @@ def index_resources_by_transaction(
             transaction_id, len(resourceids), (datetime.now() - start).seconds
         )
     )
+
+
+def index_resources_by_time(
+    start_time,
+    batch_size=settings.BULK_IMPORT_BATCH_SIZE,
+    quiet=False,
+    use_multiprocessing=False,
+    max_subprocesses=0,
+    recalculate_descriptors=False,
+):
+    """
+    Indexes all the resources with a transaction id
+
+    Keyword Arguments:
+    quiet -- Silences the status bar output during certain operations, use in celery operations for example
+    recalculate_descriptors - forces the primary descriptors to be recalculated before (re)indexing
+
+    """
+    start = datetime.now()
+
+    with connection.cursor() as cursor:
+        cursor.execute(
+            """SELECT DISTINCT resourceinstanceid FROM edit_log WHERE timestamp >= %s;""",
```

Comment:
I may be misreading the logic here, but won't this attempt to re/index resources that have been deleted?

Reply:
That's a great point; I think it will reindex those resources.  So this needs to be more sensitive to the edittype of delete.

Reply:
Given the isolated nature of this feature, I think it makes sense just to get this merged in before the feature freeze and fix the issue that @whatisgalen has identified in a separate PR.